src/scss/form/button.scss

```diff
@@ -40,6 +40,8 @@
 .button.blank:not(.disabled):hover,
 .button.blank:not(.disabled).hover { background: var(--color-shape-highlight-medium); }
 
+.pageMainDate .button.active { background: var(--color-shape-highlight-medium); }
```

Comment:
Refactoring: take class from css/popup/confirm, move it to form/button and add `.active` class the same way
```.button.blank:not(.disabled):hover, .button.blank:not(.disabled).hover```